xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/Logic.js

```diff
@@ -1245,15 +1284,12 @@ define('xwiki-livedata', [
       });
     },
 
-
-
-
+    getEditEntryProperty(source, entryId, propertyId) {
+      return liveDataSource.getEditEntryProperty(source, entryId, propertyId);
```

Comment:
As mentioned previously, it doesn't have to go through the live data source. This needs to be a configuration of the HTML displayer.